### DISCUSSION TITLE
Is it possible to simulate multiple DUs/CUs?

### DISCUSSION THREAD

**Discussion by amendoza-laude** [srsran/srsRAN_Project#810] [General Help] [answered]
I am particularly interested in understanding whether it is possible to generate and manage multiple DUs and CUs within srsRAN. We are not interested in mobility functionality, so we don't care about handovers. Specifically, I would like to know if there is any kind of documentation about this topic, as I haven't been able to find anything yet.

Thanks,

**Comment by dvdgrgrtt** [comment, marked as answer]
Hi!

You can have one CU with multiple DUs following the [handover tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/handover/source/index.html).

You ca also run multiple instances of the gNB, each of them with one CU and one or more DUs - just be careful to have dedicated configuration files and change them so that each gNB has different IDs and different network addresses.

Hope this helps.

**Reply by Ali78k** [reply]
Hi @dvdgrgrtt . Is it possible to do this HO scenario with ZMQ?

**Reply by dvdgrgrtt** [reply]
Yes, with some work. Have a look at [this tutorial](https://docs.srsran.com/projects/4g/en/latest/app_notes/source/handover/source/index.html) - it refers to the old LTE code, but the GNU-Radio broker that allows the UE to reach two base stations can be used also with the new NR gNB.

**Reply by dvdgrgrtt** [reply]
Also, have a look at [this other thread](https://github.com/srsran/srsRAN_Project/discussions/815).

**Reply by Ali78k** [reply]
Many thanks @dvdgrgrtt . I'll try it soon.
